origins_workflow/src/Controller/ModerationStateController.php

```diff
@@ -69,6 +92,15 @@ public function changeState($nid, $new_state) {
           '@user' => $this->currentUser()->getAccountName(),
         ]);
         $this->logger->notice($message);
+
+        if (!empty(\Drupal::request()->query->get('confirm'))) {
+          $message = t('Moderation state of "@title" changed to @new_state', [
+            '@title' => $entity->getTitle(),
+            '@new_state' => $new_state_entity->label(),
```

Comment:
Getting the moderation state object to allow me to pull the label out for it required some mental gymnastic here. You essentially need to use a service to pull the plugin class which in turn allows you to load the state object. Phew